test_chan/test_chan/__main__.py

```diff
@@ -0,0 +1,227 @@
+import time
```

Comment:
will this get picked up by circle, or do we need to manually run it?

Reply:
It should get picked up automatically by circle.